BREAKING.md

```diff
@@ -44,6 +45,10 @@ These have been moved to the `@fluid-internal` scope and are no longer published
 
 `IFluidHTMLView`, `ReactViewAdapter`, and `HTMLViewAdapter` were deprecated in 2.0.0-internal.3.2.0, and are now removed.
 
+### IFluidTokenProvider removed
+
+The IFluidTokenProvider interface was deprecated in 2.0.0-internal.3.2.0, and is now removed.
```

Comment:
Do we have any guidance for users of `IFluidTokenProvider`?

Reply:
I don't expect there are any - and I would recommend against it.  If one does exist they could fork the interface for their own use (we don't use/require it) but I wouldn't recommend.